src/highdicom/sr/templates.py

```diff
@@ -3759,8 +4078,8 @@ def __init__(
         language_of_content_item_and_descendants: Optional[
             LanguageOfContentItemAndDescendants
         ] = None,
-        image_library_groups: Optional[
-            Sequence[ImageLibraryEntryDescriptors]
+        referenced_images: Optional[
+            Sequence[Dataset]
```

Comment:
Since this will be a breaking change, we should add a note to the corresponding section of the documentation.